qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.test_framework import MINIMAL_SC_HEIGHT, MINER_REWARD_POST_H200
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_false, assert_true, assert_equal, initialize_chain_clean, \
+    start_nodes, start_node, sync_blocks, sync_mempools, connect_nodes_bi, mark_logs, \
```

Comment:
Remove unused imports.

```suggestion
    start_nodes, connect_nodes_bi, mark_logs, \
```